src/problems/RadDust/test_rad_dust.cpp

```diff
@@ -74,23 +74,14 @@ template <> AMREX_GPU_HOST_DEVICE auto RadSystem<DustProblem>::ComputeFluxMeanOp
 	return ComputePlanckOpacity(rho, Tgas);
 }
 
-template <>
-AMREX_GPU_HOST_DEVICE auto RadSystem<DustProblem>::ComputeThermalRadiation(amrex::Real temperature, amrex::GpuArray<double, nGroups_ + 1> const &boundaries)
-    -> quokka::valarray<amrex::Real, nGroups_>
+template <> AMREX_GPU_HOST_DEVICE auto RadSystem<DustProblem>::ComputeThermalRadiationSingleGroup(amrex::Real temperature) -> amrex::Real
 {
-	auto radEnergyFractions = ComputePlanckEnergyFractions(boundaries, temperature);
-	const double power = radiation_constant_ * temperature;
-	auto Erad_g = power * radEnergyFractions;
-	return Erad_g;
+	return radiation_constant_ * temperature;
```

Comment:
The notation here is confusing. Is temperature really T^4? If so, please rename the variable to that. If not, can you clarify why the source term is only radiation_constant * T and not T^4?

Reply:
I have to assume B = a_rad * T_d in this test in order to linearize this problem so that I can obtain an analytic solution. I added a comment to explain this. The motivation of doing this is also explained before Eq. 44 of the overleaf document quokka-thermochemistry.